README.md

```diff
@@ -1,8 +1,236 @@
 # Soccer Vision 3D RViz Markers
 
-[![Build and Test (foxy)](../../actions/workflows/build_and_test_foxy.yaml/badge.svg?branch=rolling)](../../actions/workflows/build_and_test_foxy.yaml?query=branch:rolling)
-[![Build and Test (galactic)](../../actions/workflows/build_and_test_galactic.yaml/badge.svg?branch=rolling)](../../actions/workflows/build_and_test_galactic.yaml?query=branch:rolling)
-[![Build and Test (humble)](../../actions/workflows/build_and_test_humble.yaml/badge.svg?branch=rolling)](../../actions/workflows/build_and_test_humble.yaml?query=branch:rolling)
-[![Build and Test (rolling)](../../actions/workflows/build_and_test_rolling.yaml/badge.svg?branch=rolling)](../../actions/workflows/build_and_test_rolling.yaml?query=branch:rolling)
+[![Build and Test (rolling)](https://github.com/ros-sports/soccer_vision_3d_rviz_markers/actions/workflows/build_and_test_rolling.yaml/badge.svg?branch=rolling)](https://github.com/ros-sports/soccer_vision_3d_rviz_markers/actions/workflows/build_and_test_rolling.yaml?query=branch:rolling)
 
 Package that contains libraries to display [soccer_vision_3d_msgs](https://index.ros.org/p/soccer_vision_3d_msgs/) as RViz Markers.
+
+The node simply listens for ball, field boundary, goalposts, markings, obstacles and robots,
+converts them to visualization markers and publishes them so they can be visualized in RViz.
+
```

Comment:
Link to documentation, if applicable and available.

Reply:
I just set up a new docs directory, and I will start writing up documentation there:

https://github.com/ros-sports/soccer_vision_3d_rviz_markers-docs/issues/1